#!/usr/bin/env python
PACKAGE="map_filter"
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("mean_k", int_t, 0, "setMeanK", 100, 1, 500)
gen.add("stddev_mul_thresh", double_t, 0, "setStddevMulThresh", 1.0, 0.0, 50.0)

exit(gen.generate(PACKAGE, "filter_node", "Filter"))